README.md

```diff
@@ -3,9 +3,15 @@ Snowplow to Redshift data processing.
 
 ## Run
 
-Since this code is containerized to run on Lambda, the main entrypoint isn't quite designed to run locally.
-However, we can build the Docker image and run it locally, then ping it to try it out locally. We can, of course,
-still test all the functions used by the main handler and run those separately (e.g. in a scratch file) if so desired.
+The code is primarily intended to be run on AWS Lambda, and there is a backfill CLI when needed.
+
+### Backfill
+To run a backfill, make sure you have AWS credentials on your machine and that you have any MFA commands run already
+as well (e.g. `aws-mfa`). Provide the target database's credentials via environment variables and run the backfill
+script for the intended partner site, a given date, and the number of days leading up to said date.
```

Comment:
🔧 Should this be "starting from" instead of "leading up," given the example run below?